warehouse/integrations/deltalake/deltalake.go

```diff
@@ -1070,35 +1070,30 @@ func (*Deltalake) DownloadIdentityRules(*misc.GZipWriter) (err error) {
 	return
 }
 
-// GetTotalCountInTable returns total count in tables.
-func (dl *Deltalake) GetTotalCountInTable(ctx context.Context, tableName string) (total int64, err error) {
-	fetchTotalCountExecTime := dl.Stats.NewTaggedStat("warehouse.deltalake.grpcExecTime", stats.TimerType, stats.Tags{
```

Comment:
Stats are not required now. We added these during POC for the python connector. Removing this now.